src/tg_bot_float_db_app/misc/exceptions.py

```
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class BotDbException(Exception):
```

1. Usually custom exceptions have constructors with parameters which indicate how these exceptions should be instantiated.
2. In our case, I'd go with only one constructor parameter called `message`. The callers, when they want to raise this exception, will need to prepare a message and then pass it to the exception's constructor.

---

Let's call the file bot_db_exception.py